### PR TITLE
Remove has_key

### PR DESCRIPTION
**Pull request by benjaminjkraft**
Welcome to the wonderful world of Python 2.3.  Apparently `in` is not only more
pythonic than `has_key`, it's also
[faster](http://stackoverflow.com/a/1323880/904174).  Generated mostly with
codemod, although in some cases I manually replaced it with `get`.

### CODE REVIEW

File: esp/esp/program/modules/handlers/teachereventsmodule.py [hunk 1]
```diff
@@ -181,7 +181,7 @@ def teacher_events(self, request, tl, one, two, module, extra, prog):
                     # decide type
                     type = "training"
 
-                    if data.has_key('submit') and data['submit'] == "Add Interview":
+                    if 'submit' in data and data['submit'] == "Add Interview":
```
Comment (mgersh) [inline esp/esp/program/modules/handlers/teachereventsmodule.py]: Here's one more that can be changed to a get()

File: esp/esp/users/views/usersearch.py [hunk 2]
```diff
@@ -139,8 +135,7 @@ def get_user_list(request, listDict2, extra=''):
         keys = request.POST['keys'].split(',,')
 
         for key in keys:
-            if request.POST.has_key('operator_'+key) and \
-               request.POST['operator_'+key]         and \
+            if request.POST.get('operator_'+key) and \
```
Comment (mgersh) [inline esp/esp/users/views/usersearch.py]: And this one actually wanted to be `in`

Comment (benjaminjkraft) [inline esp/esp/users/views/usersearch.py]: Why?

Comment (mgersh) [inline esp/esp/users/views/usersearch.py]: Oh, whoops, I misread the diff on this one, you're right.

I was continuing my thought from the other comment several files up the page! I dunno, it made sense in my head.

Comment (benjaminjkraft) [inline esp/esp/users/views/usersearch.py]: Oh, I think github was being dumb and not showing me that one, whoops.

**Pull request merged**